app/src/main/res/values-vi/strings.xml

```diff
@@ -1,3 +1,251 @@
+    <string name="please_click_back_again_to_exit">Nhấn nút BACK lần nữa để thoát!</string>
+    <string name="email">Email</string>
+    <string name="password">Mật khẩu</string>
+    <string name="new_password">Mật khẩu mới</string>
+    <string name="current_password">Mật khẩu hiện tại</string>
+    <string name="confirm_password">Xác nhận mật khẩu</string>
+    <string name="forgot_password">Quên mật khẩu?</string>
+    <string name="log_in">Đăng nhập</string>
+    <string name="is_empty">Vui lòng nhập trường này !</string>
+    <string name="invalid_email_format">Email không đúng định dạng !</string>
+    <string name="least_6_characters">Vui lòng nhập ít nhất 6 ký tự !</string>
+    <string name="confirm_password_does_not_match">Xác nhận mật khẩu không khớp !</string>
+    <string name="request_leave">Yêu cầu đi trễ, về sớm</string>
+
+    <string name="employee_name">Tên nhân viên</string>
+    <string name="employee_code">Mã nhân viên</string>
+    <string name="project_name">Tên dự án</string>
+    <string name="position_name">Vị trí công việc</string>
+    <string name="branch">Chi nhánh</string>
+    <string name="group">Phòng ban</string>
+    <string name="contract_date"> Ngày bắt đầu làm việc chính thức</string>
+    <string name="start_probation_date">Ngày bắt đầu thử việc</string>
+    <string name="end_probation_date">Ngày kết thúc thử việc</string>
+    <string name="staff_type">Loại nhân viên</string>
+    <string name="position">Vị trí</string>
+    <string name="gender">Giới tính</string>
+    <string name="birthday">Ngày sinh</string>
+    <string name="off_have_salary_company_pay">Nghỉ có lương - Công ty trả</string>
+    <string name="number_day_off">Số ngày nghỉ</string>
+    <string name="off_no_salary">Nghỉ không lương</string>
+
+    <string name="off_type">Hình thức nghỉ</string>
+    <string name="annual_leave">Nghỉ phép thường niên (14.5 ngày)</string>
+    <string name="leave_for_marriage">Kết hôn (3.0 ngày)</string>
+    <string name="leave_for_child_marriage">Con kết hôn (1.0 ngày)</string>
+    <string name="funeral_leave">Nghỉ lễ tang (3.0 ngày)</string>
+    <string name="leave_for_care_of_sick_child">Con ốm (20.0 ngày)</string>
+    <string name="pregnancy_examination_leave">Nghỉ khám thai (5.0 ngày)</string>
+    <string name="sick_leave">Nghỉ đau ốm (60.0 ngày)</string>
+    <string name="miscarriage_leave">Nghỉ sẩy thai (50.0 ngày)</string>
+    <string name="maternity_leave">Nghỉ thai sản (180.0 ngày)</string>
+    <string name="wife_labor_leave">Vợ sinh con (14.0 ngày)</string>
+    <string name="company_pay">Công ty trả</string>
+    <string name="insurance_coverage">Bảo hiểm trả</string>
+    <string name="select_date">Chọn ngày</string>
+    <string name="select_time">Chọn thời gian</string>
+    <string name="reason">Lý do</string>
+    <string name="create">Tạo mới</string>
+    <string name="created_at">Thời gian tạo</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.created_at` appears to be unused
 Unused resources make applications larger and slow down builds.